feat: add KubeCon EU 2025 blog post and update announcement bar (#952)
open-feature/openfeature.dev: blog/2025-02-03-kubecon-eu-25.md

---
slug: "kubeconeu25"
title: "OpenFeature at KubeCon EU '25"

> date: 2025-02-03

Reviewer: ```suggestion
date: 2025-02-06
```

authors: [beeme1mr]
image: /img/blog/2024-09-19-kubecon-na-24/logo.png
description: "OpenFeature at KubeCon EU '25"
tags: [kubecon, cncf, session, talk, booth, "OpenFeature summit", "co-located"]
draft: false
---

> We are excited to announce that the OpenFeature Summit will be returning to KubeCon + CloudNativeCon Europe 2025!

Reviewer: ```suggestion
We are excited to announce the OpenFeature Summit will be returning to KubeCon + CloudNativeCon Europe 2025!
```

> We are excited to announce that the OpenFeature Summit will be returning to KubeCon + CloudNativeCon Europe 2025!

Reviewer: "We are excited to announce" 

A LOT of our posts start with this exact sentence, I suggest an alternative.

Reviewer: "Great news! The OpenFeature summit..."

"Guess who's back? Once again the OpenFeature summit..."

Or ask an LLM?

> We are excited to announce that the OpenFeature Summit will be returning to KubeCon + CloudNativeCon Europe 2025!

Author: ```suggestion
Great news! The OpenFeature summit the OpenFeature Summit will be returning to KubeCon + CloudNativeCon Europe 2025.
```

  - **Date:** Wednesday, April 2, 2025
  - **Time:** 11:15 - 11:45 BST
  - **Location:** Level 3 | ICC Capital Suite 10-12

- [How We Progressively Deliver Changes To Kubernetes Using Canary Deployments and Feature Flags](https://kccnceu2025.sched.com/event/1txAN/how-we-progressively-deliver-changes-to-kubernetes-using-canary-deployments-and-feature-flags-bob-walker-octopus-deploy) - Bob Walker, Octopus Deploy
  - **Date:** Thursday, April 3, 2025
  - **Time:** 16:00 - 16:30 BST
  - **Location:** Level 0 | ICC Capital Hall | Room 2

- [Type-safe Feature Flagging in OpenFeature: Lessons Learned From Using Feature Flags at Google](https://kccnceu2025.sched.com/event/1txHF/type-safe-feature-flagging-in-openfeature-lessons-learned-from-using-feature-flags-at-google-michael-beemer-dynatrace-florin-mihai-anghel-google?iframe=yes&w=100%&sidebar=yes&bg=no) - Michael Beemer, Dynatrace & Florin-Mihai Anghel, Google
  - **Date:** Friday, April 4, 2025
  - **Time:** 11:45 - 12:15 BST
  - **Location:** Level 1 | Hall Entrance N10 | Room H

### Visit Our Booth

> Don't miss the chance to visit our booth at the Solutions Showcase!

Reviewer: Should we say "project pavilion" instead?

> Don't miss the chance to visit our booth at the Solutions Showcase!

Author: ```suggestion
Don't miss the chance to visit our booth in the project pavilion!
```
